package.json

```diff
@@ -50,9 +50,7 @@
       "**/*.node"
     ],
     "files": [
-      "**/*",
-      "!docs${/*}",
-      "!node_modules/@paulcbetts/cld/deps/cld${/*}"
+      "app/**/*"
     ],
```

Comment:
@andersk we added this because the spellchecker module is generating a lot of unnecessary binary files which we don't need in the app. I don't know the current status but this was the case then. The app size will increase if we remove this.

Reply:
We haven’t had `@paulcbetts/cld` since 107e5229141879b69225153dc9e6d15f54c8c7bf. On further investigation, it was renamed to `cld`. Fixed in master.